resolver/src/main/java/io/netty/resolver/InetNameResolver.java

```diff
@@ -40,32 +37,15 @@
      */
     protected InetNameResolver(EventExecutor executor) {
         super(executor);
-        loopbackAddress = SocketUtils.loopbackAddress();
     }
 
     /**
+     * @deprecated Use {@link SocketUtils#loopbackAddress()}.
```

Comment:
https://github.com/netty/netty/pull/6334 is pushing in the direction of marking the packages as `@UnstableApi` ... I like this. Unless you have strong objections I will follow that approach here. We can also discuss a followup PR to put in the `@UnstableApi` on the remainder of the package.